home/migrations/0010_cachesettings.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.1.12 on 2021-06-16 12:23
```

Comment:
its good to minimize number of migration files - so if you have couple of them, just revert all created migrations (`manage.py migrate home 0009`), delete all newly created migrations and re-create it as a single migration file